js/interface.js

```diff
@@ -156,7 +152,8 @@ Fliplet.Widget.register('com.fliplet.theme', function() {
                 method: 'POST',
                 url: 'v1/widget-instances?appId=' + Fliplet.Env.get('appId'),
                 data: {
-                    widgetId: widgetInstanceId
+                  widgetId: widgetInstanceId,
```

Comment:
can you rename this variable from `widgetInstanceId` to `widgetId`? I think it's misleading